OrbitQt/ProcessItemModel.h

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_QT_PROCESS_ITEM_MODEL_H_
+#define ORBIT_QT_PROCESS_ITEM_MODEL_H_
+
+#include <QAbstractItemModel>
+#include <QVariant>
+
+#include "process.pb.h"
+
+namespace OrbitQt {
+
+class ProcessItemModel : public QAbstractItemModel {
+  // Q_OBJECT
```

Comment:
This macro should be there. Is there a reason for commenting it?

Reply:
No, mistake on my side